tests/libc/enc/enc.c

```diff
@@ -93,6 +129,6 @@ OE_SET_ENCLAVE_SGX(
     1,    /* ProductID */
     1,    /* SecurityVersion */
     true, /* AllowDebug */
-    4096, /* HeapPageCount */
-    1024, /* StackPageCount */
+    512,  /* HeapPageCount */
```

Comment:
I have this vague recollection that these were increased as a work-around, if that memory is true, then this PR resolves the actual problem and the work-around goes away. Can you confirm I understand that correctly?

Reply:
These parameters are unique for each test. When Mike optimized the libc test, I believe he reduced the TCS count to 2. 
Since there is no EPC paging on Windows and we have limited EPC memory on Windows ACC VMs, we need to reduce the enclave size to run on Windows.